src/model/Char_model.py

```diff
@@ -0,0 +1,75 @@
+import itertools
+
+from src.classes.model_result import ModelResult
+from src.classes.text_part import TextPart
+from src.model.standard_model import StandardModel
+
+
+class CharModel(StandardModel):
+
+    def predict(self, text: str, min_p: float = 0.01) -> ModelResult:
+        """
+        main function of this class, genetates predictions
+        :param text: text from the user .example :  "ויב?א ה את הש??ם ואת ה?רץ"
+        :param min_p: minimal value of prediction score
+        :return: charModelResult , an object encapsulating list of TextParts
+        """
+        sm = StandardModel('Embible/tavbert-50-epochs')
```

Comment:
1. you don't have to create a new object of `standard model`, you are already extending it.
2.  dont use `'Embible/tavbert-50-epochs'` as hard coded value, you should yse the path passed on the constructor.
    we maybe want in the future (probebly not but still...) to use a different char model path